phoenix5-spark/README.md

```diff
@@ -337,13 +401,13 @@ df.show
 
 ### Load as an RDD, using a Zookeeper URL
 ```scala
+import org.apache.spark.SparkConf
 import org.apache.spark.SparkContext
 import org.apache.spark.sql.SQLContext
 import org.apache.phoenix.spark._
 import org.apache.spark.rdd.RDD
 
-val sparkConf = new SparkConf().set("spark.ui.showConsoleProgress", "false")
-val sc = new SparkContext("local", "phoenix-test", sparkConf)
+val sc = new SparkContext("local", "phoenix-test")
```

Comment:
Nit:
You removed the showConsoleProgress setting here, but kept it below.
TBH I don't know what that does, but shouldn't it be consistent ?

Reply:
From spark doc, this flag is to enable/desable some logs in console.
Since this param has no effect on connector behavior I removed it in the other examples.

Reply:
thx